spotbugs/src/main/java/edu/umd/cs/findbugs/ba/jsr305/TypeQualifierResolver.java

```diff
@@ -68,7 +68,8 @@ public class TypeQualifierResolver {
     };
 
     private static final ClassDescriptor[] NULLABLE_CLASS_DESCRIPTORS = new ClassDescriptor[] {
-        DescriptorFactory.createClassDescriptor("edu/umd/cs/findbugs/annotations/Nullable"),
+        // intentionally ignored, semantics is different to other Nullable annotations, see #3243
```

Comment:
Can you please add a link to the issue as well or at least mention before the issue number that it's an issue and not a PR?

Reply:
Not sure I understood, #3243 is an issue, not a PR and the comment refers to the issue.

Reply:
I meant to either update like this:
```suggestion
        // intentionally ignored, semantics is different to other Nullable annotations, see issue #3243
```
Or something like this:
```suggestion
        // intentionally ignored, semantics is different to other Nullable annotations, see <a href="https://github.com/spotbugs/spotbugs/issues/3243">#3243</a>
```

As from the number alone in the code it's not 100% clear whether it's an issue or a PR.